docs/3.1. Discovery - Registered Operation.md

```diff
@@ -4,7 +4,35 @@ _(c) AMWA 2016, CC Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)_
 
 This document describes usage of NMOS APIs for discovery in cases where where a distributed registry is available.
 
-## Registration
+## Registration API
+
+### DNS-SD Advertisement
+
+The preferred method of Registration API advertisement is via unicast DNS. Registration APIs MUST additionally be capable of producing an mDNS advertisement of the type \_nmos-register.\_tcp. This MAY be disabled via a user-configurable method.
+
+Registration APIs supporting versions 'v1.2' and below MUST additionally be capable of producing an mDNS advertisement of type \_nmos-registration.\_tcp. This MAY be disabled via a user-configurable method.
```

Comment:
"MUST additionally"... doesn't that imply a v1.2 API must produce both `_nmos-register._tcp` and `_nmos-registration._tcp`? Ah, no, this is talking about a v1.3 implementation that _also_ supports v1.2. Got it. 👍